website_anchor_smooth_scroll/static/src/js/website_anchor_smooth_scroll.js

```diff
@@ -2,25 +2,25 @@
  * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). */
 odoo.define('website_anchor_smooth_scroll.website_anchor_smooth_scroll', function (require) {
 
-"use strict";
+    "use strict";
 
-function website_anchor_smooth_scroll(event) {
-    event.preventDefault();
-    var target = $(event.currentTarget.hash);
+    function website_anchor_smooth_scroll (event) {
+        event.preventDefault();
+        var target = $(event.currentTarget.hash);
 
-    return $('html, body')
-    .stop()
-    .animate({
-        'scrollTop': target.offset().top - 100
-    })
-    .promise()
-    .done(function(element) {
-        history.pushState(null, document.title, event.target.hash);
-    });
-}
+        return $('html, body')
+            .stop()
+            .animate({
+                'scrollTop': target.offset().top - 100,
+            })
+            .promise()
+            .done(function (element) {
```

Comment:
Oh sorry, I didn't notice that comment.

You can remove it safely:

```suggestion
            .done(function () {
```

In JS, only the arguments to the left are required. The rest of them can be obtained from the `arguments` builtin. Since `element` is the rightmost argument and it is not used, you can omit it.

Reply:
thanks. done